src/handlers/browser/launch.ts

```diff
@@ -180,17 +184,29 @@ export const launchBrowser = async (
         log.log(`k6 Studio extension loaded`, response)
       } catch (error) {
         // If we fail to load the extension, we'll log the error and continue without it.
-        log.error('Failed to start browser recording:', error)
+        log.error('Failed to start browser recording: ', error)
+
+        browserWindow.webContents.send(BrowserHandler.Error, {
+          reason: 'extension-load',
+          fatal: false,
+        })
```

Comment:
At this point, the browser is already starting so instead of closing it again I opted to send a non-fatal toast to the UI.